app/components/detail-terms-list.js

```diff
@@ -82,7 +72,7 @@ export default Component.extend({
    *
    * @property filteredTerms
    * @type {Ember.computed}
-   * @public
+   * @protected
```

Comment:
This might be internal, but there is no reason it couldn't be used outside of this component.  All that is to say I don't think it should be marked private.